scripts/linters/other_files_linter.py

```diff
@@ -190,6 +191,13 @@ def check_third_party_libs_type_defs(
 
                 if lib_version[0] == '^':
                     lib_version = lib_version[1:]
+                # In cases where the version is in the form of git commit hashes
+                # such as 'git+https://github.com/username/repo#commit-hash',
+                # we extract the commit hash and use it as the version.
+                elif lib_version.startswith('git+https://github.com/'):
```

Comment:
Can you modify `GIT_COMMIT_HASH_REGEX` so you can check `git+https://github.com/' too?

Reply:
@gp201 will that be required? as we are checking this `git+https://github.com/` within the elif condition too?

Reply:
Could we just collapse it into just one condition?

Modify the regex just how Praneeth suggested and use it as a condition in elif?
`elif re.search(GIT_COMMIT_HASH_REGEX, lib_version):`

Reply:
done @aasiffaizal @gp201